app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    cmd, file1, file2 = command.split()
+    if cmd == "mv" and file1 != file2:
+        path_to_file = file2.split("/")
+        if len(path_to_file) > 1:
+            name_dir = path_to_file[0]
+            os.mkdir(name_dir)
+            for i in range(1, len(path_to_file) - 1):
+                name_dir = name_dir + "/" + path_to_file[i]
+                os.mkdir(name_dir)
+        with open(file1, "r") as file_in, \
+             open(file2, "w") as file_out:
```

Comment:
```suggestion
        with (
            open(file1, "r") as file_in,
            open(file2, "w") as file_out
        ):   
```

Reply:
Done

Reply:
The first solution has verification files, the second one has these files removed